subsys/net/ip/Kconfig

```diff
@@ -447,6 +447,37 @@ config NET_TCP_RECV_QUEUE_TIMEOUT
 	  SEQ 2. But if we receive SEQs 5,4,3,7 then the SEQ 7 is discarded
 	  because the list would not be sequential as number 6 is be missing.
 
+config NET_TCP_KEEPALIVE
+	bool "TCP Keep-Alives msg"
+	depends on NET_TCP
+	default y
+	help
+	  RFC1122#TCP Keep-Alives
+	  Enable TCP_KEEPIDLE, TCP_KEEPINTVL and TCP_KEEPCNT
+	  options processing. Note that TCP_KEEPIDLE and TCP_KEEPINTVL have to be set
+	  in seconds.
```

Comment:
Could you move this information (about seconds value) to relevant config options help text, so that it is found there together with the option itself.